index.js

```diff
@@ -77,6 +77,17 @@ const ContainerService = {
       });
 
       ss.storage.savedConfiguration = object;
+
+      // Maybe rename the Banking container.
+      if (prefService.get("privacy.userContext.enabled") !== true) {
+        let identity = ContextualIdentityService.getIdentityFromId(3);
```

Comment:
The idea here is to rename Banking to Finance in any release where containers are not already enabled.